portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,57 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+import java.util.ArrayList;
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> commentsarray = new ArrayList<String>();
```

Comment:
nit: Java convention is to name variables in lower camel case, meaning that all first letters of a word in a variable name are capitalized except for the very first letter: https://www.geeksforgeeks.org/java-naming-conventions/ So here, it'd be commentsArray